apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -1090,6 +1103,18 @@ export class WorkshopForm extends React.Component {
                 />
               )}
             </Col>
+            <Col sm={2}>
+              {this.shouldRenderModules() && (
+                <ModuleSelect
+                  course={this.state.course}
+                  module={this.state.module}
+                  readOnly={this.props.readOnly}
+                  inputStyle={this.getInputStyle()}
+                  validation={validation}
+                  onChange={this.handleModuleChange}
```

Comment:
Since this only calls the single line of `this.handleFieldChange(event);` I think this could be simplified to `onChange={handleFieldChange}` and you could remove `handleModuleChange`?